Widgets/widget.css

```diff
@@ -2,11 +2,12 @@ body {
   font-family: 'Cantarell';
   color:white;
   background-color: #194d6c;
+  font-weight:700;
```

Comment:
@Ganon11 and i will have to confirm this doesn't mess up any font stuff on our widgets